spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigMapPropertySource.java

```diff
@@ -49,34 +48,21 @@ public Fabric8ConfigMapPropertySource(KubernetesClient client, String name) {
 
 	public Fabric8ConfigMapPropertySource(KubernetesClient client, String applicationName, String namespace,
 			Environment environment) {
-		super(getName(applicationName, getNamespace(client, namespace)),
```

Comment:
I've renamed `getNamespace` to `getApplicationNamespace`, since such a method with such a name already existed in that class. This is now a proper overload. imho, this simplifies intent of these methods.